packages/fiori/src/FlexibleColumnLayout.ts

```diff
@@ -1072,6 +1072,25 @@ class FlexibleColumnLayout extends UI5Element {
 		return this.effectiveSeparatorsInfo[0].arrowVisible;
 	}
 
+	get startSeparatorValue() {
+		const startColumnWidth = this.startColumnWidth;
+		if (typeof startColumnWidth === "string" && startColumnWidth.endsWith("%")) {
+			return parseInt(startColumnWidth);
+		}
+		return 50;
```

Comment:
Let's define these values (50, 75) as constants on the top of the file and add tests in the cypress spec.